pkg/models/Models.go

```diff
@@ -12,6 +12,7 @@ type Node struct {
 	Esxi                 Esxi     `json:"esxi"`
 	ExpectedServiceNames []string `json:"expectedServiceNames"`
 	RAM                  int      `json:"RAM"`
+	CV                   bool     `json:"CV"`
```

Comment:
Since this is a bool, let's rename this to "isCV" within the struct definition. It follows this go conventions guideline https://github.com/unknwon/go-code-convention/blob/master/en-US/naming_rules.md .  
The reason for this request would be for readability

Reply:
Fixed